docs/source/notebooks/Tutorial.ipynb

```diff
@@ -1804,7 +1804,7 @@
    "source": [
     "### 1. Path encoding\n",
     "\n",
-    "To perform quantum computations using photons, we need an encoding: a correspondance between our Fock states and our qubit states.\n",
+    "To perform quantum computations using photons, we need an encoding: a correspondence between our Fock states and our qubit states.\n",
```

Comment:
"mapping" instead of "correspondence" ?

Reply:
done https://github.com/Quandela/Perceval/pull/435/commits/d667ddde294fc8d399860f33630ff7e33acfa7d8